### DISCUSSION TITLE
WordPress caught an error with one of your plugins, 12 Step Meeting List.

### DISCUSSION THREAD

**Discussion by Alanwr2024** [code4recovery/12-step-meeting-list#1598] [Q&A] [answered]
I cannot login or access meeting plugin when activated Wordpress email said this plugin is causing a fatal error. I can login in when deactivated. my hosting said contact developer the site is aameetingspahrump.org site with plugin activated but cannot get in to change anythig

**Comment by thessnake03** [comment]
Is everything up to date, WordPress and tsml? I'd do that first.

If that doesn't do it, sounds like there's a plugin conflict between tsml and another one. What else do you have active?

**Reply by Alanwr2024** [reply]
I deactivated all othe plugins still same probelm. wp is  6.7.1 up to date.  i get this Parse error: syntax error, unexpected ')' in /home/aameybix/public_html/wp-content/plugins/12-step-meeting-list/includes/admin_meeting.php on line 138  My plugins are 
12 Step Meeting List  	3.16.9 	
	
Akismet Anti-spam: Spam Protection  	5.3.5 	
	
Classic Editor  	1.6.7 	
	
Disable Comments  	2.4.6 	
	
Hello Dolly  	1.7.2 	
	
LiteSpeed Cache  	6.5.4 	
	
W3 Total Cache  	2.8.2 	
	
WordPress Importer  

copy admin_meeting.php 
![Screenshot_2024-12-21_20-10-08](https://github.com/user-attachments/assets/5e291e02-f5e5-4242-a9c2-50b05d987277)

**Reply by joshreisner** [reply]
hey thanks for the extra detail! that's an odd one since whitespace usually doesn't cause issues - we're not seeing any errors on the demo site. i wonder if you could try replacing those lines with the following:

```php
                <?php echo wp_kses(
                    sprintf(
                        // translators: %s is the list of supported conference providers
                        __('If this meeting has videoconference information, please enter the full valid URL here. Currently supported providers: %s. If other details are required, such as a password, they can be included in the Notes field above, but a ‘one tap’ experience is ideal. Passwords can be appended to phone numbers using this format <code>+12125551212,,123456789#,,#,,444444#</code>', '12-step-meeting-list'),
                        implode(', ', tsml_conference_providers()),
                    ),
                    TSML_ALLOWED_HTML
                ) ?>
```

if this fixes it for you we would definitely push this out as an update

**Reply by Alanwr2024** [reply]
it just moved the error got this now Parse error: syntax error, unexpected 'URL' (T_STRING), expecting ',' or ')' in /home/aameybix/public_html/wp-content/plugins/12-step-meeting-list/includes/admin_meeting.php on line 143
![Screenshot_2024-12-21_21-40-18](https://github.com/user-attachments/assets/15b8ecf9-e8dd-4beb-96ea-4d8796dc8a36)

**Comment by Alanwr2024** [comment]
thanks i not sure  how to edit the .php file I am running linux i got that copy from my cpanel did not see option to replace file i found the edit function made back up

here is link to backup file: https://drive.google.com/file/d/1wQ4tgQD1mNZqCv4zyuva4pWIQVuDpNck/view?usp=sharing

**Comment by Alanwr2024** [comment, marked as answer]
I fixed it, by updating my php verson from 7.2 to 8.4 thank you

**Reply by joshreisner** [reply]
ok great, thanks for the update. 

this helped me figure out the issue, which is that the trailing commas in function calls [were added in PHP 7.3](https://laravel-news.com/php-trailing-commas-functions). we'll release an [update](https://github.com/code4recovery/12-step-meeting-list/commit/233cecf9ad7b4745f3a7db20edf3bc0d12ee71b9#diff-1568cb52d908a4cbc40830eace32f4c06aefe91f8808159731f8ba586dcafdceR135) so that it works in PHP 7.2 (although WordPress recommends 7.4 and up)

**Reply by Alanwr2024** [reply]
It was by the grace of God stumbled upon this I barely know about wordpress let alone pluginin and php thank you for your support you gave me the push to keep going. your pal Alan
